### PR TITLE
spark-sstfile-generator comments reviews

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Reviewed the comments in spark sstfiles and README file.

### CODE REVIEW

File: src/tools/spark-sstfile-generator/README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
-Generate sst files from hive tables datasource, guided by a mapping file, which maps hive tables to vertexes and edges.
-Multiple vertexes or edges may map to a single hive table, where a partition column will be used to distinguish different
-vertex or edge.  
-The hive tables may be periodically be regenerated by upstream system to reflect the latest data in so far, and may be
-partitioned by a time column to indicate the time when data are generated.  
-*$HADOOP_HOME* env need to be set for running this job.  
+Guided by a mapping file, sst files are generated from hive tables datasource. A mapping file maps hive tables to vertexes and edges.
```
Comment (darionyaphet) [inline src/tools/spark-sstfile-generator/README.md]: sst is the abbreviation of `Sorted String Table` . Maybe using `SST` will be better ?

File: src/tools/spark-sstfile-generator/README.md [hunk 2]
```diff
@@ -67,9 +70,8 @@ _make sure to keep consistent with DEBUG_LEVEL when building, or there will be s
 [0x7f7c689c1747]
 ```
 
-# TODO
-1. Add database_name property to graphspace level and tag/edge level, which the latter will override the former when provided in both levels   
-2. Schema column definitions' order is important, keep it when parsing mapping file and when encoding  
+# TO DO
```
Comment (darionyaphet) [inline src/tools/spark-sstfile-generator/README.md]: why append a space ?

File: src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/MappingConfiguration.scala [hunk 3]
```diff
@@ -120,10 +120,10 @@ object Tag {
   *
   * @param tableName            hive table name
   * @param name                 edge type name
-  * @param fromForeignKeyColumn which column is srcID
-  * @param fromReferenceTag     which Tag does srcID column reference
-  * @param toForeignKeyColumn   which column is dstID
-  * @param toReferenceTag       which Tag does dstID column reference
+  * @param fromForeignKeyColumn  srcID column
```
Comment (darionyaphet) [inline src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/MappingConfiguration.scala]: please remove the extra space :)

File: src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SparkSstFileGenerator.scala [hunk 4]
```diff
@@ -15,16 +15,16 @@ import org.slf4j.LoggerFactory
...
   * The following use cases are supported:
   *
   * <p>
-  * Generate sst files from hive tables datasource, guided by a mapping file, which maps hive tables to vertexes and edges,
+  * Generate sst files from hive tables datasource, guided by a mapping file, which maps hive tables to vertexes and edges.
   * Multiple vertexes or edges may map to a single hive table, where a partition column will be used to distinguish different
   * vertex or edge.
-  * The hive tables may be periodically be regenerated by your business system to reflect the latest data in so far, and may be
-  * partitioned by a time column to indicate the time when generated.
+  * The hive tables may periodically be regenerated by your business system to reflect the latest data so far, and   be
```
Comment (darionyaphet) [inline src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SparkSstFileGenerator.scala]: extra space

File: src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SparkSstFileGenerator.scala [hunk 5]
```diff
@@ -37,7 +37,7 @@ object SparkSstFileGenerator {
   val SSF_OUTPUT_HDFS_DIR_CONF_KEY = "nebula.graph.spark.sst.file.hdfs.dir"
 
   /**
-    * cmd line's options, which's name following the convention: input will suffix with "i", output will suffix with "o"
+    * cmd line's options, whose name follow the convention: input will start with suffix "i", output with "o"
```
Comment (spacewalkman) [inline src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SparkSstFileGenerator.scala]: I really mean suffix "i",  not `start with suffix i`

Comment (Amber1990Zhang) [reply src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SparkSstFileGenerator.scala]: sorry but 'suffix' is not a verb

Comment (spacewalkman) [reply src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SparkSstFileGenerator.scala]: `start with suffix xxx` sounds weird,  People would be wondering which option name convention should be followed:
_xxx_someOption OR someOption_xxx_ (Seriously, i means the latter). 
How about put it this way:
`input options name end with 'i', while output end with 'o'`

**Comment by darionyaphet** [comment]
Basically LGTM 🎉 👍  Please fix the conflict.

File: src/tools/spark-sstfile-generator/README.md (hunk 1 above)
Comment (Amber1990Zhang) [reply src/tools/spark-sstfile-generator/README.md]: sst is acceptable

File: src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SparkSstFileGenerator.scala (hunk 5 above)
Comment (Amber1990Zhang) [reply src/tools/spark-sstfile-generator/src/main/scala/com/vesoft/tools/SparkSstFileGenerator.scala]: good idea.

**Comment by jude-zhu** [comment]
Jenkins go

**Pull request merged**